components/02-elements/form/input/_input.scss

```diff
@@ -36,4 +35,17 @@ $textarea-min-height    : 100px !default;
             min-height: $textarea-min-height;
         }
     }
+
+    &--small {
+        text-align: right;
```

Comment:
Leave it as it is, but IMO it should be added as a overwrite to minicart component, b/c isn't so reusable.